dashboard/test/ui/features/ed_programs/pd/teacher_application.feature

```diff
@@ -129,3 +129,89 @@ Scenario: Basic teacher application submission
   Then I see no difference for "Principal approval confirmation form"
   Then I close my eyes
 
+# [MEG] TODO: Remove experiment flag from the two URLs below
+Scenario: Teacher saves, re-opens, and submits an application
+  Given I create a teacher named "Severus"
+  And I am on "http://studio.code.org/pd/application/teacher?enableExperiments=teacher-application-saving-reopening"
```

Comment:
Ah, interesting question. I was imagining all of this to be launched together (saving + reopening) because the [spec](https://docs.google.com/document/d/1kbRAQ3To-MHVmz2KHnyaN6Mh6EiJk8F_zRNS1F8kSK8/edit?pli=1#heading=h.57tjrgucyls5) has them all as P0. I'll check in with Tess about it!

Reply:
@davidsbailey Checked in with Tess: assuming all of this gets launched in the next month (which I think it will), we'll keep the experiment together

Reply:
ok, sounds good! well-named experiment, then 😁